livekit-agents/livekit/agents/voice/agent.py

```diff
@@ -368,6 +369,41 @@ def _get_activity_or_raise(self) -> AgentActivity:
 
         return self._activity
 
+    def __getstate__(self) -> dict[str, Any]:
```

Comment:
How should it looks like from the user perspective when they add their own states, should they override this function and call `super().__getstate__`?